cmd/node/node.go

```diff
@@ -661,24 +657,23 @@ func (app *App) initServices(ctx context.Context,
 		pubsub.ChainGossipHandler(syncHandler, tBeacon.HandleSerializedFollowingVotingMessage))
 	app.host.Register(blocks.NewBlockProtocol, pubsub.ChainGossipHandler(syncHandler, blockListener.HandleBlock))
 	app.host.Register(activation.AtxProtocol, pubsub.ChainGossipHandler(syncHandler, atxDB.HandleGossipAtx))
-	app.host.Register(state.IncomingTxProtocol, pubsub.ChainGossipHandler(syncHandler, svm.HandleGossipTransaction))
+	app.host.Register(svm.IncomingTxProtocol, pubsub.ChainGossipHandler(syncHandler, state.HandleGossipTransaction))
 	app.host.Register(activation.PoetProofProtocol, poetListener.HandlePoetProofMessage)
 
 	app.proposalBuilder = proposalBuilder
 	app.blockListener = blockListener
 	app.mesh = msh
 	app.syncer = newSyncer
 	app.clock = clock
-	app.state = processor
+	app.svm = state
```

Comment:
This happens twice - see [line 676](https://github.com/spacemeshos/go-spacemesh/pull/2992/files#diff-69bfff7b16ff2da1ebcca3ff8dcd296aa906d17d4af33acac1b959329fcd95b0R676) below

Reply:
Thanks! Fixing this.